Update get-a-price-sheet.md

Agazoth opened the pull request:
As stated in the documentation, deflate returns compressed content. If the example has this header, the response should have a compressed result and not an uncompressed result. Removing the header fixes that.

Moving response success and error codes makes the page more readable.

opbld31 commented:
OPS Build status updates of commit _[5671d66](https://github.com/MicrosoftDocs/partner-rest/pull/6/commits/5671d66b85351f717909c02087e858c8e37d2e1b)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

opbld30 commented:
OPS Build status updates of commit _[5671d66](https://github.com/MicrosoftDocs/partner-rest/pull/6/commits/5671d66b85351f717909c02087e858c8e37d2e1b)_: 

:clock10: **Incremental building**: average incremental building time is _18 sec(s)_

opbld32 commented:
OPS Build status updates of commit _[5671d66](https://github.com/MicrosoftDocs/partner-rest/pull/6/commits/5671d66b85351f717909c02087e858c8e37d2e1b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[partner-rest/TOC.yml](https://github.com/MicrosoftDocs/partner-rest/blob/referrals/partner-rest/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/partner/develop/headers?branch=pr-en-us-6) |
[partner-rest/develop/api-authentication.md](https://github.com/MicrosoftDocs/partner-rest/blob/referrals/partner-rest/develop/api-authentication.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/partner/develop/api-authentication?branch=pr-en-us-6) |
[partner-rest/develop/create-a-referral.md](https://github.com/MicrosoftDocs/partner-rest/blob/referrals/partner-rest/develop/create-a-referral.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/partner/develop/create-a-referral?branch=pr-en-us-6) |
[partner-rest/develop/headers.md](https://github.com/MicrosoftDocs/partner-rest/blob/referrals/partner-rest/develop/headers.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/partner/develop/headers?branch=pr-en-us-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C25%5Cb005616d-f7d3-db9a-e8dc-720f489c8327%5CPullRequest%5C201902251851224818-6%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=UX%2Fd4hgrBlz5YxzqV%2BNRtvkr4VFGReyKJ4nqRofBnFI%3D&st=2019-02-25T18%3A47%3A23Z&se=2019-03-28T18%3A52%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Agazoth committed:
Update get-a-price-sheet.md

As stated in the documentation, deflate returns compressed content. If the example has this header, the response should have a compressed result and not an uncompressed result. Removing the header fixes that.

Moving response success and error codes makes the page more readable.

PRMerger13 commented:
@Agazoth : Thanks for your contribution! The author(s) have been notified to review your proposed change. @JasonWHowell

PRMerger17 commented:
@Agazoth : Thanks for your contribution! The author(s) have been notified to review your proposed change.

JasonWHowell commented:
#sign-off
